alacritty/src/logging.rs

```diff
@@ -167,7 +180,10 @@ fn create_log_message(record: &log::Record<'_>, target: &str, start: Instant) ->
 fn is_allowed_target(level: Level, target: &str) -> bool {
     match (level, log::max_level()) {
         (Level::Error, LevelFilter::Trace) | (Level::Warn, LevelFilter::Trace) => true,
-        _ => ALLOWED_TARGETS.contains(&target),
+        _ => {
+            ALLOWED_TARGETS.contains(&target)
+                || EXTRA_LOG_TARGETS.as_ref().map_or(false, |elt| elt.contains(&target))
```

Comment:
```suggestion
                || EXTRA_LOG_TARGETS.as_ref().map_or(false, |targets| targets.contains(&target))
```

Took me too long to figure out what the hell an "elt" is supposed to be lol.

Reply:
But they are on the same line...